src/components/Registration/DefineRegistrationSummary.vue

```diff
@@ -85,20 +85,18 @@
       </template>
 
       <!-- Business Number -->
-      <template v-if="isTypePartnership">
```

Comment:
Wait - why did you delete this?

Reply:
According to the [Invision design](https://projects.invisionapp.com/share/3511UPXPMT2P#/screens/464799355), SP also displays BN, so I removed this condition.

Reply:
This change is obsolete as we want to continue displaying BN only for GPs.
Resolving.